collections/_posts/2022-11-26-SparseMultiChannelVAE.md

```diff
@@ -0,0 +1,311 @@
+
+where $$\mathbb{E}_{c}$$ is the average over channels computed empirically.
+
+The following figure illustrates the underlying concept:
+
+![](/collections/images/smcvae/modeling_encoder_side.jpg)
+
+&nbsp;
+
+## Derivation of the ELBO loss
+
+The minimization of the above equation is equivalent to the maximization of the following Evidence Lower BOund - ELBO (the corresponding demonstration is given in the appendix of this post):
+
+$$\mathcal{L} = \mathbb{E}_{c}\left[\mathbb{E}_{\textbf{z} \sim q(\textbf{z} \vert \textbf{x}_c)}[log\left(p(\textbf{x} \vert \textbf{z})\right)] - D_{KL}\left( q(\textbf{z} \vert \textbf{x}_c) \parallel p(\textbf{z})\right) \right]$$
+
+* Making the hypothesis that every channels is conditionally independent from all others given $$\textbf{z}$$, we have:
+
+$$p(\textbf{x} \vert \textbf{z})=\prod_{i=1}^{C}p(\textbf{x}_i \vert \textbf{z})$$
+
+$$\mathcal{L}$$ can thus be rewritten as:
+
+$$\mathcal{L} = \mathbb{E}_{c}\left[\mathbb{E}_{\textbf{z} \sim q(\textbf{z} \vert \textbf{x}_c)}\left[\sum_{i=1}^{C}log\left(p(\textbf{x}_i \vert \textbf{z})\right)\right] - D_{KL}\left( q(\textbf{z} \vert \textbf{x}_c) \parallel p(\textbf{z})\right) \right]$$
+
+&nbsp;
+
+The minimization of $$\mathbb{E}_{c}[D_{KL}( q(\textbf{z} \vert \textbf{x}_c) \parallel p(\textbf{z} \vert \textbf{x}))]$$ is thus equivalent to the minimization of 
+
+$$\mathcal{L^{*}} = \mathbb{E}_{c}\left[D_{KL}\left( q(\textbf{z} \vert \textbf{x}_c) \parallel p(\textbf{z})\right) - \mathbb{E}_{\textbf{z} \sim q(\textbf{z} \vert \textbf{x}_c)}\left[\sum_{i=1}^{C}log\left(p(\textbf{x}_i \vert \textbf{z})\right)\right]\right]$$
+
+
+The following figure illustrates the minimization of $$\mathcal{L^{*}}$$:
+
+![](/collections/images/smcvae/core_minimization.jpg)
+
+&nbsp;
+
+![](/collections/images/smcvae/core_minimization_2_1.jpg)
+![](/collections/images/smcvae/core_minimization_2_2.jpg)
+![](/collections/images/smcvae/core_minimization_2_3.jpg)
+
+&nbsp;
+
+## Reconstruction of missing channels
+
+* The terms $$\mathbb{E}_{\textbf{z} \sim q(\textbf{z} \vert \textbf{x}_c)}\left[\sum_{i=1}^{C}log\left(p(\textbf{x}_i \vert \textbf{z})\right)\right]$$  force each channel to the joint decoding of itself and every other channel
+at the same time
+
+* This property allows to reconstruct missing channels $$\{\hat{\textbf{x}}_i\}$$ for the available ones $$\{\tilde{\textbf{x}}_j\}$$ as:
+
+$$\hat{\textbf{x}}_i = \mathbb{E}_j\left[\mathbb{E}_{\textbf{z} \sim q(\textbf{z} \vert \tilde{x}_c)}\left[\left(p(\textbf{x}_i \vert \textbf{z})\right)\right]\right]$$
+
+&nbsp;
+
+## Comparison with standard VAE
+
+* In case of $$C=1$$, the proposed sMCVAE model is similar to a classical VAE. 
+
+* sMCVAE is different from a VAE where all the channels are concatenated into a single one. In that case there cannot be missing channels if we want to infer the latent space variables
```

Comment:
Simplify the phrasing here:
```suggestion
* sMCVAE is different from a VAE where all the channels are concatenated together, since such a VAE cannot handle missing channels.
```